app/models/cart.rb

```diff
@@ -2,11 +2,16 @@ class Cart
   attr_reader :data
 
   def initialize(data)
-    @data = data || Hash.new
+    @data = data || {"" => 0}
   end
 
   def add_item(item_id)
     @data[item_id] ||= 0
     @data[item_id] += 1
   end
+
+  def cart_count
+    require 'pry'; binding.pry
```

Comment:
Remove debugger entry point `binding.pry`.<br>Do not use semicolons to terminate expressions.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.